submit/workflow/__init__.py

```diff
@@ -0,0 +1,97 @@
+"""Defines submission stages and workflows supported by this UI."""
+
+from typing import Iterable, Optional, Callable, List, Iterator
+
+from arxiv.submission.domain import Submission
+from dataclasses import dataclass
+
+from . import stages
+from .stages import Stage
```

Comment:
absolute imports may be preferable for clarity

Reply:
I was thinking that this was a place  where the stages are closely related to the workflows.